Add XGSPON ONT (#81)
hack-gpon/hack-gpon.github.io: _ont_xgs/ont-hisense-ltf7267-bha+.md

---
title: Hisense LTF7267-BHA+
has_children: false
layout: default
parent: Hisense
---

# Hardware Specifications

|             |              |
| ----------- | ------------ |
| Vendor      | Hisense      |
| Model       | LTF7267-BHA+ |
| Chipset     | Cortina      |
| Manufacter  | Hisense      |
| Flash       | 128MB        |
| RAM         | 128MB        |
| System      | Custom Linux by Cortina (Saturn SDK) based on Kernel 4.4 |
| XSGMII      | Yes          |
| Optics      | SC/APC       |
| IP address  | 192.168.0.1  |
| Web Gui     | ✅ user `admin`, password `system` |
| Telnet      | ✅ user `root`, password `hbmtsfp` |
| SSH         | ✅ user `root`, password `hbmtsfp` |
| Form Factor | miniONT SFP+  |

# External/Internal Photo

{% include image.html file="ont-hisense-ltf7267-bha+_front.jpg" alt="Hisense LTF7267-BHA+ External" caption="Hisense LTF7267-BHA+ External" %}

{% include image.html file="ont-hisense-ltf7267-bha+_inside.jpg" alt="Hisense LTF7267-BHA+ Internals" caption="Hisense LTF7267-BHA+ Internals" %}

## UART Console (115200, 8bit)

> It's near the SFP header (look at red-square on _Internals Pic_), TX and RX pads and groun need to be connected on USB2TTL adapters

Reviewer: ```suggestion
It's near the SFP header (look at the red-square on _Internals Pic_). TX, RX and ground pads need to be connected to a USB2TTL adapter supporting a logic level of 3.3V
```

## List of software versions
- 22.05.26.1 - 20220527052622 (from /etc/hi_version - /etc/version)

## List of partitions

| dev   | size     | erasesize | name            |
| ----- | -------- | --------- | --------------- |
| mtd0  | 00040000 | 00001000  | "ssb"           |
| mtd1  | 00002000 | 00001000  | "uboot-env"     |
| mtd2  | 00002000 | 00001000  | "dtb0"          |
| mtd3  | 0003c000 | 00001000  | "kernel0"       |
| mtd4  | 00300000 | 00001000  | "rootfs0"       |
| mtd5  | 004c0000 | 00001000  | "dtb1"          |
| mtd6  | 00300000 | 00001000  | "kernel1"       |
| mtd7  | 004c0000 | 00001000  | "rootfs1"       |
| mtd8  | 00001000 | 00001000  | "userdata"      |
| mtd9  | 00001000 | 00001000  | "squashfs_ubi"  |
| mtd10 | 00001000 | 00001000  | "userdata"      |

This ONT supports dual boot. 

`kernel0` and `rootfs0` respectively contain the kernel and firmware of the first image, `kernel1` and `rootfs1` the kernel and the firmware of the second one

# Useful Commands

To access Cortina Shell (needed to check OMCI stuff and XGSPON status) you can use the following command:

```sh
# app-cli
```

> This console can be also reached opening a connection to `192.168.0.1:2233`

Reviewer: ```suggestion
This console can also be reached by opening a telnet connection to `192.168.0.1:2233`
```

## Check Activation Status (from app-cli and telnet\ssh session)

> Open two sessions, one to `192.168.0.1` and one to `192.168.0.1:2233`

Reviewer: ```suggestion
Open two telnet sessions, one to `192.168.0.1` and one to `192.168.0.1:2233`
```

    |               Filter              |          Treatment          |
    |-----------------------------------|-----------------------------|
    |    Outer   |         Inner        | R |   Outer    |    Inner   |
    |------------|----------------------| E |------------|------------|
    | PRI | VID  | PRI | VID  | Filter  | M | PRI | VID  | PRI | VID  |
    |-----|------|-----|------|---------|---|-----|------|-----|------|
    |  15 | 4096 |  14 | 4096 |No filter| 3 |  15 |    0 |  15 |    0 |
    |  14 | 4096 |  14 | 4096 |No filter| 3 |  15 |    0 |  15 |    0 |
    |  15 | 4096 |  15 | 4096 |No filter| 0 |  15 | 4096 |   0 |    1 |
    |  15 | 4096 |   8 |  835 |No filter| 1 |  15 |    0 |   8 |  100 |
    |-----|------|-----|------|---------|---|-----|------|-----|------|
```

## Changing the Hardware Version

```sh
# vi /config/scfg.txt
```

Append line below to the file and save it to change HWVER

```
CHAR-ARRAY      CFG_ID_GPON_VERSION = {0x46,0x35,0x36,0x38, 0x34,0x53,0x5f,0x76, 0x31,0x00,0x00,0x00, 0x00,0x00}; ##GPON version string, default value is V1.0
```

Reboot ONT to apply the change

## Changing the Equipment ID

```sh
# vi /config/scfg.txt
```

> Append line below to the file and save it to change EQID

Reviewer: ```suggestion
Append the line below to the file and save it to change EQID
```

```
CHAR-ARRAY      CFG_ID_GPON_EQID = {0x46,0x49,0x42,0x45, 0x52,0x20,0x42,0x6f, 0x78,0x00,0x00,0x00, 0x00,0x00,0x00,0x00, 0x00,0x00,0x00,0x00}; ##GPON ME ONU2G equiment id value, default is saturn
```

Reboot ONT to apply the change

## Changing Serial Number

```sh
# vi /config/scfg.txt
```

Append lines below to the file and save it to change Serial Number

```
STRING CFG_ID_PON_VENDOR_ID = SMBS;
INT CFG_ID_PON_VSSN = 0xAABBCCDD;
```

Reboot ONT to apply the change

## Changing OLT Emulation Type

```sh
# vi /config/scfg.txt
```

> Append line below to the file and save it to change Serial Number

Reviewer: ```suggestion
Append lines below to the file and save it to change Serial Number
```

```
STRING          CFG_ID_PON_OLT_TYPE                             = ALCL; ##GPON OLT Vendor name, support ZTE,ADTRAN,ALCL,CALIX,SUMITOMO,CORTINA,HUAWEI
```

Reboot ONT to apply the change

## Changing OMCC Version

> {% include alert.html content="In Italy, if you are under Huawei OLT it's mandatory to use 0xA3, while on Alcatel 0xB4, otherwise you will get O5 status but no MIBs - Note that this can be quirk of TIM Italy" alert="Warning" icon="svg-warning" color="red" %}

Reviewer: ```suggestion
{% include alert.html content="In Italy, if you are under some Huawei OLT it's mandatory to use 0xA3, while on Alcatel 0xB4, otherwise you will get O5 status but no MIBs - Note that this can be quirk of TIM Italy" alert="Warning" icon="svg-warning" color="red" %}
```

```sh
# vi /config/scfg.txt
```

> Append line below to the file and save it to change Serial Number

Reviewer: ```suggestion
Append the line below to the file and save it to change Serial Number
```

```
CHAR            CFG_ID_OMCC_VERSION                             = 0xB2;
```

Reboot ONT to apply the change

## Changing LOID Username/Password

> {% include alert.html content="The value 0x0 is null, take note of your LoID and password from original ONT" color="red" %}

Reviewer: ```suggestion
{% include alert.html content="The value 0x0 is null, take note of your LoID and password from the original ONT" color="red" %}
```

```sh
# vi /config/scfg.txt
```

> Append line below to the file and save it to change Serial Number

Reviewer: ```suggestion
Append lines below to the file and save it to change LoID and LoID password
```
